src/App.js

```diff
@@ -108,23 +102,20 @@ function App() {
       });
   }
 
-  useEffect(() => {
-    fetch(`http://localhost:3000/days`)
-      .then((r) => r.json())
-      .then((data) =>
-        setCurrentDaysData(data.filter((dayObj) => dayObj.user_id === 1))
-      );
-  }, [rerender]);
+  const { data: fetchedDaysData, loading: daysLoading, fetchData: fetchDaysData } = useCustomFetch(
+    'http://localhost:3000/days'
```

Comment:
`nit`: Please no magic strings.

Reply:
Is this a good edit to the above?

```
 const daysEndpoint = 'http://localhost:3000/days';

  const { data: isDaysDataFetched, loading: daysLoading, fetchData: fetchDaysData } = useCustomFetch(daysEndpoint);
```

Also, I'm still getting the hang of pull requests, is there a better way to respond with code edits, or does that require a new pull request?